src/components/com_tjdashboard/administrator/views/dashboard/tmpl/default.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * @version    CVS: 1.0.0
+ * @package    Com_Tjdashboard
+ * @author     Techjoomla <extensions@techjoomla.com>
+ * @copyright  2017 Techjoomla
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+// No direct access
+defined('_JEXEC') or die;
+
+
+$document = JFactory::getDocument();
+$script  = 'var root_url = "' . Juri::root() . '";';
```

Comment:
@monikaatvowel Create Js constant in the component entry file. Also, change var to const and use camelcase variables.